pkg/cloudprovider/providers/gce/gce_healthchecks.go

```diff
@@ -17,11 +17,31 @@ limitations under the License.
 package gce
 
 import (
+	"flag"
+	"fmt"
 	"time"
 
+	"k8s.io/kubernetes/pkg/api/v1"
+	"k8s.io/kubernetes/pkg/master/ports"
+	utilversion "k8s.io/kubernetes/pkg/util/version"
+
+	"github.com/golang/glog"
 	compute "google.golang.org/api/compute/v1"
 )
 
+const (
+	minNodesHealthCheckVersion = "1.7.0"
+	nodesHealthCheckPath       = "/healthz"
+)
+
+var (
+	lbNodesHealthCheckPortFlag int
+)
+
+func init() {
+	flag.IntVar(&lbNodesHealthCheckPortFlag, "cloud-provider-gce-lb-nodes-healthcheck-port", ports.ProxyHealthzPort, "Port opened on nodes for LB health checks")
```

Comment:
Seems like the wrong place for this flag, as it is not a property of GCE. Is it not possible to pass in the port when the health check is created?